src/main/java/org/codinjutsu/tools/jenkins/logic/JenkinsJsonParser.java

```diff
@@ -166,10 +184,37 @@ private Build getBuild(@Nullable JsonObject lastBuildObject) {
         if (duration != null) {
             builder.duration(duration);
         }
-
+        // set parameter
+        Optional.ofNullable(getActions(lastBuildObject))
+                .flatMap(actions -> actions.stream()
+                        .filter(action -> isContainParameters((JsonObject) action))
+                        .findFirst()
+                )
+                .ifPresent(action ->
+                        builder.buildParameterList(getBuildParameters((JsonObject) action))
+                );
         return builder.build();
     }
 
+    @Nullable
+    private JsonArray getActions(@NotNull JsonObject lastBuildObject) {
+        return lastBuildObject.getCollection(createJsonKey(ACTIONS));
+    }
+
+    @NotNull
+    private List<BuildParameter> getBuildParameters(JsonObject action) {
+        return action.getCollection(createJsonKey(PARAMETERS)).stream()
+                .map(parameter -> BuildParameter.of(
+                        ((JsonObject) parameter).getString(createJsonKey("name")),
+                        ((JsonObject) parameter).getString(createJsonKey("value"))
+                ))
```

Comment:
got exception here for password field because no value is present:
```
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'value' of org/codinjutsu/tools/jenkins/model/BuildParameter.of must not be null
	at org.codinjutsu.tools.jenkins.model.BuildParameter.$$$reportNull$$$0(BuildParameter.java)
	at org.codinjutsu.tools.jenkins.model.BuildParameter.of(BuildParameter.java)
	at org.codinjutsu.tools.jenkins.logic.JenkinsJsonParser.lambda$getBuildParameters$3(JenkinsJsonParser.java:207)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at org.codinjutsu.tools.jenkins.logic.JenkinsJsonParser.getBuildParameters(JenkinsJsonParser.java:211)
	at org.codinjutsu.tools.jenkins.logic.JenkinsJsonParser.lambda$getBuild$2(JenkinsJsonParser.java:194)
	at java.base/java.util.Optional.ifPresent(Optional.java:183)
	at org.codinjutsu.tools.jenkins.logic.JenkinsJsonParser.getBuild(JenkinsJsonParser.java:193)
	at org.codinjutsu.tools.jenkins.logic.JenkinsJsonParser.getLastBuild(JenkinsJsonParser.java:367)
	at java.base/java.util.Optional.map(Optional.java:265)
	at org.codinjutsu.tools.jenkins.logic.JenkinsJsonParser.getJob(JenkinsJsonParser.java:278)
	at org.codinjutsu.tools.jenkins.logic.JenkinsJsonParser.createJob(JenkinsJsonParser.java:132)
	at org.codinjutsu.tools.jenkins.logic.RequestManager.loadJob(RequestManager.java:231)
	at org.codinjutsu.tools.jenkins.logic.RequestManager.loadFavoriteJobs(RequestManager.java:326)
	at org.codinjutsu.tools.jenkins.view.BrowserPanel$LoadSelectedViewJob.loadJobs(BrowserPanel.java:497)
	at org.codinjutsu.tools.jenkins.view.BrowserPanel$LoadSelectedViewJob.run(BrowserPanel.java:472)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:962)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:472)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:236)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:178)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:658)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:610)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:165)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:236)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)

```